TestSwitchApi/TestSwitchApi/DataMigrations/002_AlterTable.sql

```diff
@@ -0,0 +1,26 @@
+﻿ALTER TABLE public."Tests"
+  ADD COLUMN "Title" character varying (128),
+  ADD COLUMN  "Question" text,
+  ADD COLUMN "Input" character varying (128),
+  ADD COLUMN "ExpectedOutput" character varying (128);
+
+ALTER TABLE public."Tests" DROP COLUMN "ModelAnswer";
+
+ALTER TABLE public."Tests" ADD COLUMN "ModelAnswer" text;
```

Comment:
I still think we just don't want this yet.
I'd rather add this table when we might actually use it - could we leave this out for now?

Reply:
removed